aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -108,14 +107,34 @@ class QLinearInt8 final : public torch::OperatorKernel {
     fbgemm::DoNothing<> doNothingObj{};
 
     const int32_t* bias_ptr = nullptr;
-    if (bias.has_value()) {
-      Tensor bias_vec = bias.value();
-      TORCH_CHECK(bias_vec.dim() == 1, "bias should be a vector (1D Tensor)");
+    at::Tensor qbias;
+    if (pack_ptr.bias.has_value()) {
+      at::Tensor bias = pack_ptr.bias.value();
+      // Temporary: Quantize bias
```

Comment:
Will you consider to avoid requantizing bias every time for better performance? Caching and compare the scale/zp of the input tensor should work fine.

Reply:
I think this is just a temporary solution and it will be replaced. The plan is to quantize bias dynamically as a post operation step after linear/conv. cc @dskhudia

Reply:
Bias will need to be requantized every time in eager mode, this is because we do not know the input tensor's scale and zero-point ahead of time. When we have a static graph, we can optimize this further.